src/mobile/src/main/java/tupperdate/android/appbars/MainBottomBar.kt

```diff
@@ -0,0 +1,55 @@
+package tupperdate.android.appbars
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.vectorResource
+import tupperdate.android.R
+import tupperdate.android.home.BarButton
+import tupperdate.android.home.littleButtonSize
+import tupperdate.android.home.normalButtonSize
+import tupperdate.android.ui.dislikeButton
+import tupperdate.android.ui.likeButton
+import tupperdate.android.ui.recipeAddButton
+import tupperdate.android.ui.returnButton
+
+@Composable
+fun MainBottomBar(
```

Comment:
I think we could find a better name than bottom bar, especially since it's closely related to the recipe card.
Also, we could move that file to the home package

Reply:
Ok